File: z_cs_py_yapilanlar/CConfigManager.py
from datetime import datetime

class CConfigManager:
    def __init__(self):
        self.son_okuma_zamani = None
        self.son_yazma_zamani = None
        self.son_reset_zamani = None
        self.son_clear_zamani = None
        self.son_okuma_zamani_key = None
        self.son_yazma_zamani_key = None
        self.son_reset_zamani_key = None
        self.son_clear_zamani_key = None

    def __del__(self):
        pass

    def create_key(self, sistem, suffix):
        return f"{sistem.Name};{sistem.Sembol};{sistem.Periyot};{suffix}"

    def set_value(self, sistem, key, value):
        self.son_yazma_zamani_key = f"{key};YAZMA ZAMANI"
        self.son_yazma_zamani = datetime.now().strftime("%H:%M:%S")
        sistem.nesne_kaydet(self.son_yazma_zamani_key, self.son_yazma_zamani)
        sistem.nesne_kaydet(key, value)

    def get_value(self, sistem, key):
        self.son_okuma_zamani_key = f"{key};OKUMA ZAMANI"
        self.son_okuma_zamani = datetime.now().strftime("%H:%M:%S")
        sistem.nesne_kaydet(self.son_okuma_zamani_key, self.son_okuma_zamani)
        return sistem.nesne_getir(key)

File: z_cs_py_yapilanlar/CConfigManager.cs
public class CConfigManager
{
    string SonOkumaZamani;
    string SonYazmaZamani;
    string SonResetZamani;
    string SonClearZamani;

    string SonOkumaZamaniKey;
    string SonYazmaZamaniKey;
    string SonResetZamaniKey;
    string SonClearZamaniKey;

    ~CConfigManager()
    {

    }

    public CConfigManager()
    {

    }

    public string CreateKey(dynamic Sistem, string Suffix)
    {
        return ( Sistem.Name + ";" + Sistem.Sembol + ";" + Sistem.Periyot + ";" + Suffix );
    }

    public void SetValue(dynamic Sistem, string Key, dynamic Value)
    {
        SonYazmaZamaniKey = Key + ";" + "YAZMA ZAMANI";
        SonYazmaZamani = DateTime.Now.ToString("HH:mm:ss"); //yyyy.MM.dd HH:mm:ss
        Sistem.NesneKaydet(SonYazmaZamaniKey, SonYazmaZamani);

        Sistem.NesneKaydet(Key, Value);
    }

    public dynamic GetValue(dynamic Sistem, string Key)
    {
        SonOkumaZamaniKey = Key + ";" + "OKUMA ZAMANI";
        SonOkumaZamani = DateTime.Now.ToString("HH:mm:ss"); //yyyy.MM.dd HH:mm:ss
        Sistem.NesneKaydet(SonOkumaZamaniKey, SonOkumaZamani);

        return Sistem.NesneGetir(Key);
    }
}